clients.md

```diff
@@ -26,35 +25,59 @@ Basing on best practices.
+4. At the end of, e.g., two weeks—the exact period depends on a project—we collect all the closed tasks and form a clear invoice based on them. 
+5. As we have adopted a strict quality assurance approach, you end up with a well-documented and working piece of your product. Any other software development team will not need any time to put the code into further work—code speaks for itself.
 
-### Fixed Time, Fixed Budget & Flex Scope
+So you basically have transparency during the whole process and can tweak even on the lowest level. You do not lose control over your budget; you don't pay for anything but the delivered work, and since there is no long-term commitment, you can safely quit at any time.
 
-So in order to overcome the disadvantages of waterfall model, the Fixed Time, Fixed Budget, Flex Scope approach was introduced, which we believe to be the best choice for the projects you cannot leave to float freely on Scrum or other Agile rails. It was widely popularized by Basecamp developers in their ["Getting Real" (2006)](https://gettingreal.37signals.com/) bestseller book.
+Widely popularized FFF model, developed and described by Basecamp team in their "Getting Real" (2006) bestseller book, tricks you with an expectation of a full product being delivered in a fixed time; in our case, you know exactly what is going to be flexed, so you are not going to have hard time thinking about what to pare down. You just watch your product being delivered, having each piece added in an a self-containing, MVP way.
 
-To let you have the gist of it, one could say it discovers the benefits of possibility that you get a project done on-time and on-budget but... with some features missing.
+#### Audit and maintenance
 
-It might sound frightening at first, but practically it means that all parties care about bringing on a viable and effective product on time, and all the additional thoughts, non-critical issues and nonessential features are left for the next rounds for the sake of on-time and addressing delivery.
+Audit and, e.g., corporate training are, of course, special cases when we need to have the total scope to be clearly defined, so they are managed in a different way.
 
-### Time & Material
+As far as the scope is fixed, using our rule of thumb we need to have its due time to be flexible.
 
-It is plain and, moreover, one of the simplest charge models. You get as much as you pay. This can be beneficial for projects with a high level of uncertainty, when a client is ready to invest their time, steep deeply into the scope and require considerable transparency.
+We provide audit services only on place: we do not create reports that are separate from your code. We audit your repo directly, making changes and adding documentation and comments to your code. So we need to have a way to influence the processes, otherwise this audit is doomed to be practically useless for you.
```

Comment:
Not just repo, but also: full pipeline (CI, CD), documentation, deployment tools, etc.